common/src/test/kotlin/com/hartwig/actin/molecular/evidence/matching/HotspotMatchingTest.kt

```diff
@@ -1,43 +1,87 @@
+private val KNOWN_HOTSPOT = TestServeKnownFactory.hotspotBuilder().gene("gene 1").chromosome("12").position(10).ref("A").alt("T").build()
+
+private val VARIANT_ANNOTATION =
+    TestServeMolecularFactory.createVariantAnnotation(gene = "gene 1", chromosome = "12", position = 10, ref = "A", alt = "T")
+private val VARIANT_ANNOTATION2 =
+    TestServeMolecularFactory.createVariantAnnotation(gene = "gene 1", chromosome = "12", position = 11, ref = "C", alt = "G")
+private val ACTIONABLE_HOTSPOT = ImmutableActionableHotspot.builder()
+    .from(TestServeMolecularFactory.createActionableEvent())
+    .addVariants(VARIANT_ANNOTATION, VARIANT_ANNOTATION2)
+    .build()
 
 private val MATCHING_CRITERIA =
     VariantMatchCriteria(gene = "gene 1", chromosome = "12", position = 10, ref = "A", alt = "T", isReportable = true)
+private val MATCHING_CRITERIA2 =
+    VariantMatchCriteria(gene = "gene 1", chromosome = "12", position = 11, ref = "C", alt = "G", isReportable = true)
 
 class HotspotMatchingTest {
 
     @Test
-    fun `Should return true on matching hotspot`() {
-        assertThat(HotspotMatching.isMatch(HOTSPOT, MATCHING_CRITERIA)).isTrue()
+    fun `Should return true on matching known hotspot`() {
+        assertThat(HotspotMatching.isMatch(KNOWN_HOTSPOT, MATCHING_CRITERIA)).isTrue()
+    }
+
+    @Test
+    fun `Should return false on non-matching gene to known hotspot`() {
+        assertThat(HotspotMatching.isMatch(KNOWN_HOTSPOT, MATCHING_CRITERIA.copy(gene = "gene 2"))).isFalse()
+    }
+
+    @Test
+    fun `Should return false on non-matching chromosome to known hotspot`() {
+        assertThat(HotspotMatching.isMatch(KNOWN_HOTSPOT, MATCHING_CRITERIA.copy(chromosome = "13"))).isFalse()
+    }
+
+    @Test
+    fun `Should return false on non-matching position to known hotspot`() {
+        assertThat(HotspotMatching.isMatch(KNOWN_HOTSPOT, MATCHING_CRITERIA.copy(position = 12))).isFalse()
+    }
+
+    @Test
+    fun `Should return false on non-matching ref to known hotspot`() {
+        assertThat(HotspotMatching.isMatch(KNOWN_HOTSPOT, MATCHING_CRITERIA.copy(ref = "C"))).isFalse()
+    }
+
+    @Test
+    fun `Should return false on non-matching alt to known hotspot`() {
+        assertThat(HotspotMatching.isMatch(KNOWN_HOTSPOT, MATCHING_CRITERIA.copy(alt = "G"))).isFalse()
+    }
+
+    @Test
+    fun `Should return true on matching to actionable hotspot`() {
+        assertThat(HotspotMatching.isMatch(ACTIONABLE_HOTSPOT, MATCHING_CRITERIA)).isTrue()
+        assertThat(HotspotMatching.isMatch(ACTIONABLE_HOTSPOT, MATCHING_CRITERIA2)).isTrue()
     }
 
     @Test
-    fun `Should return false on non-matching gene`() {
-        assertThat(HotspotMatching.isMatch(HOTSPOT, MATCHING_CRITERIA.copy(gene = "gene 2"))).isFalse()
+    fun `Should return false on non-matching gene to actionable hotspot`() {
+        assertThat(HotspotMatching.isMatch(ACTIONABLE_HOTSPOT, MATCHING_CRITERIA.copy(gene = "gene 2"))).isFalse()
```

Comment:
It would be a little less explicit, but do you think this test class would be easier to read if the known and actionable tests were combined? E.g.
```suggestion
    fun `Should return false on non-matching gene`() {
        listOf(KNOWN_HOTSPOT, ACTIONABLE_HOTSPOT).forEach { hotspot ->
            assertThat(HotspotMatching.isMatch(hotspot, MATCHING_CRITERIA.copy(gene = "gene 2"))).isFalse()
        }
```